crates/matrix-sdk-ui/src/timeline/controller/mod.rs

```diff
@@ -586,7 +586,7 @@ impl<P: RoomDataProvider, D: Decryptor> TimelineController<P, D> {
         &self,
     ) -> (
         Vector<Arc<TimelineItem>>,
-        impl Stream<Item = VectorDiff<Arc<TimelineItem>>> + SendOutsideWasm,
+        impl Stream<Item = VectorDiff<Arc<TimelineItem>>> + SendOutsideWasm + use<P, D>,
```

Comment:
Can't we change the `impl Stream` to `VectorSubscriberStream`? Having a named type would help removing the `use<P, D>` I think, thought?